branchLayoutImpl/src/main/java/com/virtuslab/branchlayout/impl/readwrite/BranchLayoutFileReader.java

```diff
@@ -155,10 +159,17 @@ private IBranchLayoutEntry createEntry(String line, List<IBranchLayoutEntry> sub
       String line = lines.get(realLineNumber);
       if (line.isBlank()) {
         // Can't use lambda because `realLineNumber` is not effectively final
-        LOG.debug("Line no ${realLineNumber} is blank. Skipping");
+        LOG.debug("Line no ${realLineNumber + 1} is blank. Skipping");
         continue;
       }
 
+      if (!BranchLayoutFileUtils.hasProperIndentationCharacter(line, indentSpec.getIndentCharacter())) {
+        LOG.error("Line no ${realLineNumber + 1} has unexpected indentation character inconsistent with previous one");
+        throw new BranchLayoutException(realLineNumber + 1,
+            "Line no ${realLineNumber + 1} in branch layout file (${path.toAbsolutePath()}) has unexpected indentation "
```

Comment:
What if one line is indented with, say, a multiple of 4 spaces, and another is indented with a N spaces where N is not a multiply of 4?

Reply:
Isn't it `read_givenFileWithIndentWidthNotAMultiplicityOfLevelWidth_throwsException`?

Reply:
Then exception is thrown (from `getIndentLevelMethod` from line 205)